opentelemetry-stdout/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- *Breaking* time fields, `StartTime` and `EndTime` is printed on aggregation (Sum, Gauge, Histogram, ExpoHistogram) with 2 tabs, previously it was on aggregation data point, with 3 tabs, see [#2377](https://github.com/open-telemetry/opentelemetry-rust/pull/2377) and [#2411](https://github.com/open-telemetry/opentelemetry-rust/pull/2411).
```

Comment:
I think this is okay to skipp changelog for stdout

Reply:
I think it's important to mention this... I mean, my company use similar approach in production (have custom exporter that exports metrics as logs using `log` instead `println`). So imagine that someone might relay on the format of `stdout` exporter.

Reply:
> So I imagine that someone might rely on the format of stdout exporter.

I don't think so. It's explicitly called out that stdout is *only* for learning purposes, and its output format *should not* be relied upon for anything.

https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-stdout/src/lib.rs#L1-L4

(Not a blocker for this PR, just sharing that stdout is not meant for that purpose)